templates/types/extractor/fastapi/app/ui/components/extractor.py

```diff
@@ -1,29 +1,51 @@
 import reflex as rx
-from app.services.extractor import ExtractorService
-
-from .schema_editor import SchemaState
+from app.services.model import DEFAULT_MODEL
+from app.services.extractor import ExtractorService, InvalidModelCode
 
 
 class StructureQuery(rx.State):
     query: str
     response: str
     loading: bool = False
+    code: str = DEFAULT_MODEL
```

Comment:
@marcusschiesser are we going to allow editing the code instead of the JSON schema for now? Then, once Pydantic v2 is out, we can do another update, right?

Reply:
Btw, just use exec() is simple and good enough for the app (as playground to extract the data). We can later safely run it in another thread and dump and load the object from disk (using [pickle](https://docs.python.org/3/library/pickle.html) or [dill](https://github.com/uqfoundation/dill))

Reply:
yes, to not be blocked by the Pydantic v2 release, we'll do it like that for now.